charms/kfp-viz/src/components/pebble_components.py

```diff
@@ -0,0 +1,28 @@
+import logging
```

Comment:
```suggestion
# Copyright 2023 Canonical Ltd.
# See LICENSE file for licensing details.
import logging
```

Reply:
integration of the rock can be addressed in a separate PR, WDYT?

Reply:
Yup, that's alright.